tests/integration/helpers.test.js

```diff
@@ -126,13 +127,14 @@ describe('PUT /helpers/{_id}', () => {
   describe('Other roles', () => {
```

Comment:
pour le test should update referent of helpers il faudrait une vérif que l'update est bien fait

Reply:
done